magik-checks/src/main/java/nl/ramsolutions/sw/checks/ModuleDefCheck.java

```diff
@@ -148,4 +151,31 @@ protected void addIssue(
   protected void addFileIssue(final String message) {
     this.issueCollector.addFileIssue(message);
   }
+
+  /**
+   * Get all module name nodes.
+   *
+   * @param node Top node.
+   * @return a {@link Set} of {@link AstNode} objects representing module names.
+   */
+  public List<AstNode> getModuleNameNodes(final AstNode node) {
+    if (node == null) {
+      return List.of();
+    }
+    return node.getDescendants(ModuleDefinitionGrammar.MODULE_NAME);
```

Comment:
Empty line after the if statement, as it "disrupts" the regular flow (i.e., returns early from the method).